Server/Services/RevolutionaryDiscordBotService.cs

```diff
@@ -652,29 +666,37 @@ private SlashCommandBuilder BuildLanguageCommand()
                 "Displays current language translation statistics for Thrive.");
     }
 
-    private async Task HandleDsucCommand(SocketSlashCommand command)
+    private async Task HandleDaysSinceCommand(SocketSlashCommand command)
     {
         if (!await CheckCanRunAgain(command))
             return;
 
-        WatchedKeyword? underWaterCiv = await database.WatchedKeywords.FindAsync("underWaterCiv");
+        WatchedKeyword? keyword = await database.WatchedKeywords.FindAsync(
+            command.Data.Options.First().Value);
```

Comment:
Does this handle no parameter given correctly?

Reply:
I have it set up so they can only enter valid values. Otherwise, it won't let them send the command. So it should be impossible for them to send no parameter.

Reply:
Discord is still an external system that could send us garbage we have no control over.

Reply:
I'll add a try-catch then.

Reply:
Well I think maybe defaulting to underwater here would be fine? Like I said some time ago that it should be the default value.